src/constants.js

```diff
@@ -6,5 +6,8 @@ export default {
     copy: {
```

Comment:
All voucher validity now comes from the API. See https://github.com/neontribe/ARCVService/pull/102